### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by billeast**
format changes

**ATTENTION:** This template provides structure for 
- a general pull request and 
- a specific pull request in scope of IFC4.3 validation work within IFC Infrastructure Extension Deployment project.
Please fill in the corresponding section(s) and delete the rest.

# General pull request

Fixes # . *reference the issue fixed through this pull request*

## Changes and improvements proposed by this pull request
- *list all functions / files / changes you have made*
- 

*Use `@[username]` to gain attention by potential reviewers.*

# IFC4.3 Unit test

## Unit test
*provide a short description*

## Review requested
- *List all functions/concepts/entities you have added/changed/improved*
- 

@pjanck

## Link to external documentation
*optional*

## Instructions
- provide a descriptive title
> Added new UT <UnitTestName> with supporting documentation
> Expanded UT <UnitTestName> with ...
> Corrected / Amended / Updated ...
- prepend the title with `[UT] <title>` to enable easier management
- mark as WIP if work in progress
> WIP: [UT] <title>
- label the pull request with `infra depl`
- please set at least @pjanck as a reviewer and add others as needed
- delete this section

### CODE REVIEW

File: IFC 2x3/Duplex Apartment/README.md [hunk 1]
```diff
@@ -2,16 +2,21 @@
 **Dataset Duplex Apartment**
```
Comment (pjanck) [inline IFC 2x3/Duplex Apartment/README.md:2]:
```suggestion
# Dataset Duplex Apartment
```

File: IFC 2x3/Duplex Apartment/README.md [hunk 2]
```diff
@@ -2,16 +2,21 @@
...
 
 Description:
```
Comment (pjanck) [inline IFC 2x3/Duplex Apartment/README.md:4]:
```suggestion
**Description:**
```

**Review by pjanck — CHANGES_REQUESTED** [review]
And similarly as for `description` for other "sections". I think that in this way the readme would be easily readable.

**Review by billeast — COMMENTED** [review]
format updates made as requested

**Pull request merged**